src/components/Form.jsx

```diff
@@ -26,98 +28,167 @@ export default function Form() {
     setMonth('')
     setSize('')
     setPrice('')
+    setVisibleCard('nameInput')
   }
-  const handleSubmit = (event) => {
-    event.preventDefault()
+  const handleSubmit = () => {
     if (name && date && month) {
       dispatch(productAdded({ id: uuid(), name, date, month, size, price }))
-      dispatch(drawerIsOpened(false))
-      resetForm()
+      setVisibleCard('success')
     }
   }
 
+  const handleCloseClick = () => {
+    dispatch(drawerIsOpened(false))
+    resetForm()
+  }
+
   useEffect(() => {
     !isDrawerOpen && resetForm()
   }, [isDrawerOpen])
 
   return (
-    <FormStyled onSubmit={handleSubmit}>
-      <label htmlFor="name">1. Add product name:</label>
-      <input
-        onChange={(event) => setName(event.target.value)}
-        value={name}
-        type="text"
-        id="name"
-        maxLength="40"
-        autoFocus
-        placeholder="E.g. Nivea face cream"
-      />
-      {name.length >= 40 && (
-        <p>The product name can consist of up to 40 characters.</p>
-      )}
-
-      <label htmlFor="date">2. When did you open the product?</label>
-      <input
-        onChange={(event) => setDate(event.target.value)}
-        value={date}
-        type="date"
-        min="2018-01-01"
-        max={currentDate}
-        id="date"
-      />
-
-      <label htmlFor="month">
-        3. In how many months does the product expire?
-      </label>
-      <ContainerStyled>
+    <FormStyled onSubmit={(event) => event.preventDefault()}>
+      <Card isVisible={visibleCard === 'nameInput'}>
+        <label htmlFor="name">1. Add product name:</label>
+        <input
+          onChange={(event) => setName(event.target.value)}
+          value={name}
+          type="text"
+          id="name"
+          maxLength="40"
+          autoFocus
+          placeholder="E.g. Nivea face cream"
+        />
+        {name.length >= 40 && (
+          <p>The product name can consist of up to 40 characters.</p>
+        )}
+
+        <Button text="Next" onClick={() => setVisibleCard('dateInput')}/>
+        <div className="step-indicator">
+          <Circle className="active"/>
+          <Circle/>
+          <Circle/>
+          <Circle/>
+        </div>
```

Comment:
Eine kleine Idee, damit du dich mit dem step-indicator nicht immer so ausführlich wiederholen musst. Du könntest eine StepIndicator component machen in der du definierst, dass es insgesamt 4 Kreise gibt und der component ein prop wie z.B. current oder so mitgibst. Den Kreisen könntest du dann z.B. Nummern von 0-3 geben. 
Dann kannst du dein div ab Zeile 67 einfach durch die StepIndicator component ersetzten, der du dann wieder den `className="active"` übergibst und das "current" prop, in dem dann der jeweilige Kreis angesprochen wird (z.B. für den product name input `current=0`)

Reply:
Echt gute Idee! Werde ich gleich umsetzen! 👩‍💻